### ISSUE TITLE
[AP 0.6.3] Multiworld fails to generate if min_scrap_chance is greater than 0

### ISSUE THREAD

**Issue by IThisGuyHere** [T0r1nn/APLC#14]
In the latest Archipelago release, multiworlds that enable scrapsanity fail to generate if min_scrap_chance is greater than 0 and accessibility is set to 'full'. This did not happen in 0.6.2 and prior releases.

**Comment by IThisGuyHere** [comment]
It turns out that the crash is intended (see APCore [#3787](https://github.com/ArchipelagoMW/Archipelago/pull/3787)). Full accessibility should guarantee that all locations are reachable, but when none of the moons a scrap or monster spawns on meet the minimum chance, that scrap or monster *becomes* unreachable.
